src/i18n/soviet.coffee

```diff
@@ -0,0 +1,33 @@
+messages =
+
+  "settings.common": "Общее"
+  "settings.common.awayPhp": "Выезжать в заграничные командировки без проверок КГБ"
+  "settings.common.awayPhp.tooltip": "Переходить по внешним ссылкам в обход <strong>away.php</strong>"
+
+  "settings.messages": "Телеграммы и материалы"
+  "settings.messages.darkerUnread": "Новые телеграммы на более тёмной подложке"
+  "settings.messages.noPagePreviews": "Не подшивать ссылки на страницы как материалы к телеграммам"
+
+  "settings.sideMenu": "Боковое меню"
+  "settings.sideMenu.fixPosition": "Закрепить положение меню, не прокручивать из вида"
+  "settings.sideMenu.showTime": "Показывать текущее время под меню"
+
+  "settings.friends": "Товарищи"
+  "settings.friends.noPeopleYouMightKnow": "Скрывать блок \"Возможные друзья\""
+  "settings.friends.onlineByDefault": "Показывать только товарищей на связи по умолчанию"
+
+  "settings.photos": "Фотокарточки"
+  "settings.photos.noLikeOverlay": "Убрать большую кнопку \"Одобряю\" поверх фотокарточек"
+
+  "settings.videos": "Видеокаталог"
+  "settings.videos.myVideosByDefault": "Показывать только мои киноленты по умолчанию"
+
+  "settings.help": "Справочная"
+  "settings.help.community": "Клуб ВК-Х"
+  "settings.help.reportBug": "Жалобная книга"
+  "settings.help.resetSettings": "Сбросить настройки"
+
+module.exports =
+  languageId: 777
+  locale: "ru"
+  messages: messages
```

Comment:
Please add an empty line at the end of file and `push --force` to the same branch.
Otherwise solid! :+1: